apis/nucleus/src/utils/handlers/hypercube-handlers.js

```diff
@@ -0,0 +1,102 @@
+import utils from '@nebula.js/conversion/src/utils';
+import DataPropertyHandler from './data-property-handlers';
+import { getHyperCube, setFieldProperties } from './handlers-helpers';
+
+class HyperCubeHandler extends DataPropertyHandler {
```

Comment:
The class source is in sense-client [hypercube-handler.ts](https://github.com/qlik-trial/sense-client/blob/main/web/assets/objects/utils/data-properties/hypercube-handler.js)